frontend/views/containers/chatroom/file-attachment/ChatAttachmentPreview.vue

```diff
@@ -25,6 +25,7 @@
           :src='objectURLList[entryIndex]'
           :alt='entry.name'
           @click='openImageViewer(objectURLList[entryIndex])'
+          @load='onImageLoad(objectURLList[entryIndex])'
```

Comment:
Above this line is:

```
template(v-if='isForDownload')
  .c-attachment-preview(
    v-for='(entry, entryIndex) in attachmentList'
    :key='entryIndex'
    class='is-download-item'
    tabindex='0'
  )
```

`o3-mini-high` says:

> Using the array index as the key works but may cause issues if the attachment list changes frequently (for example, when items are added or removed).
• If each attachment has a unique identifier (for instance a file ID or manifestCid), consider using that as the key.

I don't know if this needs to be changed, but if it's easy to grab the manifestCid from `entry` feel free to use that as `:key`.

Reply:
Good catch. updated.